pages/cloud/nutanix/23-ovh-gateway-doc/guide.de-de.md

```diff
@@ -8,22 +8,30 @@ section: Netzwerk und Sicherheit
 order: 03
 ---
 
-**Last updated 11th January 2022**
+**Last updated 2nd May 2022**
 
 ## Objective
 
-OVHgateway is the name of the outgoing point of your cluster to the internet.
+"OVHgateway" is the name of the outgoing point of your cluster to the internet.
 
-**This guide will explain how this gateway works.**
+**This guide will explain how this gateway works, and how to redeploy it.**
 
 ## Instructions
 
 ### Technical details
 
 #### General information
 
-The VM is based on Ubuntu 18.04.5 LTS (Bionic).
-OVHgateway has a lightweight design, using 2 NICs, 1 vCPU, 1 GB memory and a 11 GiB disk.
+The VM is based on Ubuntu 20.04 LTS (“The Focal Fossa”).
+
+> [!primary]
+> The gateway is built on the basis *Daily Build* cloud images of Ubuntu.
```

Comment:
Missing "of" here:
"The gateway is built on the basis **of** *Daily Build* cloud images of Ubuntu."